### PR TITLE
Add performance pipeline.

### PR DESCRIPTION
**Pull request by rjweeks**
# Motivation and Context
Add pipeline to run the census-rm-performance-tests

# How to test?
Manually trigger pipeline to run tests against census-rm-performance GCP project.

# Links
https://github.com/ONSdigital/census-rm-performance-tests

### CODE REVIEW

**Comment by rjweeks** [comment]
[Performance test: Create Environment and Pipeline (8)](https://trello.com/c/SDKzPIeR/457-performance-test-create-environment-and-pipeline-8)

File: pipelines/performance-tests-pipeline.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+---
+resources:
+
+- name: 24h
+  type: time
+  source:
+    interval: 24h
+    days: [Monday, Tuesday, Wednesday, Thursday, Friday]
```
Comment (AdamHawtin) [inline pipelines/performance-tests-pipeline.yaml]: This resource isn't doing anything now

Comment (rjweeks) [reply pipelines/performance-tests-pipeline.yaml]: Removed.

**Review by AdamHawtin — CHANGES_REQUESTED** [review]
Needs adding to the pipeline flying pipeline.

**Review by AdamHawtin — APPROVED** [review]

**Review by davidmort — APPROVED** [review]

**Pull request merged**